app/helpers/bento_helper.rb

```diff
@@ -0,0 +1,9 @@
+module BentoHelper
+  def box_enabled?(box)
+    session[:boxes].include?(box)
+  end
+
+  def box_width
+    (12 / session[:boxes].count).to_i
```

Comment:
Definitely. 100% of this UI is going to be rewritten so I haven't been worried to much about it.